osu.Game/Overlays/OverlayRulesetSelector.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.UserInterface;
+using osu.Game.Graphics;
+using osu.Game.Rulesets;
+using osuTK;
+using osuTK.Graphics;
+using System.Linq;
+
+namespace osu.Game.Overlays
+{
+    public class OverlayRulesetSelector : RulesetSelector
```

Comment:
this class is non-abstract but not being used anywhere directly except for a test. is this planned to be used somewhere in the future?

Reply:
Will be used in `RankingsOverlay`